nemoguardrails/library/gliner/actions.py

```
@@ -129,13 +142,13 @@ async def gliner_detect_pii(
 
     try:
         total_entities = gliner_response.get("total_entities", 0)
-        return total_entities > 0
+        return _pii_detection_outcome(total_entities > 0)
     except (KeyError, TypeError) as e:
         raise ValueError(f"Invalid response from GLiNER service: {str(e)}")
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)